Gemfile

```diff
@@ -1,44 +1,44 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.4'
+gem 'rails', '~> 4.2.4'
```

Comment:
The styleguide says to pin the exact version of the framework (Rails isn't strictly semver, and does sometimes have breaking changes in patch versions).